File: api/v2010/participant/read-example-1/read-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

participants = @client.conferences('CFbbe4632a3c49700934481addd5ce1659')
                      .participants
                      .list

participants.each do |record|
  puts record.call_sid
end

File: api/v2010/participant/read-example-1/read-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$participants = $twilio->conferences("CFbbe4632a3c49700934481addd5ce1659")
                       ->participants
                       ->read();

foreach ($participants as $record) {
    print($record->callSid);
}